spec/requests/security_activity_spec.rb

```diff
@@ -1,6 +1,19 @@
 RSpec.describe "security activities" do
   let(:user) { FactoryBot.create(:user) }
 
+  context "registering a new user" do
+    let(:registration_state) { FactoryBot.create(:registration_state, :finished) }
+
+    it "records USER_CREATED events" do
+      # Stub the Registration State to sneak past redirect safeguards
+      registration_state_factory = RegistrationState.find(registration_state.id)
+      allow(RegistrationState).to receive(:find).with(nil).and_return(registration_state_factory)
```

Comment:
Ah, this can actually just be `allow(RegistrationState).to receive(:find).with(nil).and_return(registration_state)` now

Reply:
Have added this here: https://github.com/alphagov/govuk-account-manager-prototype/pull/571/commits/abb8daab5c3fe15a6bfdea2d38d83a9a9d81c384